etc/kayobe/ansible/build-ofed.yml

```diff
@@ -0,0 +1,111 @@
+---
+- name: Build OFED packages
+  hosts: ofed-builder
+  gather_facts: false
+  vars:
+    stackhpc_mlnx_ofed_file_string: MLNX_OFED_LINUX-{{ stackhpc_pulp_mlnx_ofed_version }}-rhel9.{{ stackhpc_pulp_repo_rocky_9_minor_version }}-x86_64
+  tasks:
+    - name: Install package dependencies
+      ansible.builtin.dnf:
+        name:
+          - std
+          - debootstrap
+          - kpartx
+          - cloud-init
+          - perl
+          - rpm-build
+          - automake
+          - patch
+          - kernel-devel
+          - autoconf
+          - pciutils
+          - kernel-rpm-macros
+          - lsof
+          - libtool
+          - tk
+          - gcc-gfortran
+          - tcl
+      state: present
+
+    - name: Install python dependencies
+      ansible.builtin.pip:
+        name: pulp-cli
+
+    - name: Create build directory
+      ansible.builtin.file:
+        path: /opt/ofed
+        state: directory
+        mode: 0777
+      become: true
+
+    - name: Download MellanoxOFED archive
+      ansible.builtin.get_url:
+        url: https://content.mellanox.com/ofed/MLNX_OFED-{{ stackhpc_pulp_mlnx_ofed_version }}/{{ stackhpc_mlnx_ofed_file_string }}.tgz
+        dest: /opt/ofed/ofed-archive
```

Comment:
.tgz?

Reply:
Seems to be the correct URL

Reply:
Should the dest have a .tgz suffix?

Reply:
Added, it was working without it though. Does look more normal this way

Reply:
Yeah, it's just nice to keep the extension for clarity